enhancements/monitoring/hypershift-monitoring.md

```diff
@@ -0,0 +1,392 @@
+
+3. [MonitoringStack](https://github.com/openshift/enhancements/pull/866) project offer similar functionality as UWM, but it aims for more features, flexibility and scalability due to different support models (outside of OpenShift Core) and less tight integration to OpenShift itself. It’s being developed in https://github.com/rhobs/monitoring-stack-operator. We plan to add support to [the agent mode of Prometheus](https://twitter.com/bwplotka/status/1454146102240944134) for efficient metric forwarding.
+4. A similar pipeline can be used for a portion of HSRE owned components living in each data plane.
+
+#### Customer Monitoring on Data Plane
+
+As per goals and non-goals we aim to give the CUS ability to scrape data related to their worklodads / hardware / Kubernetes state metrics that is on their data plane. As mentioned in [Non Goals](#non-goals) we don’t want to give or sell anything for scalable, centralized monitoring to customers (yet). Not beyond anything else w provide which is an opinionated UWM, CMO can deploy on normal OCP.
+
+What we have to do, is to enable and recommend them using existing workflows, which simply means providing the same data using the same protocols (Prometheus exposition format).
+
+The typical workflows:
+* HPA/VPA using Prometheus Adapter
+* Monitoring and alerting on kube-state metrics, node exporter, cadvisor and application metrics
+* Local console access with OCP metrics and dashboard
+
+We don’t see any blockers for the proposed model if we would follow the same path as GKE, EKS or any other Managed Kubernetes with hidden control plane offers. By design, HyperShift offers managed Kube API. This means that any metrics related to the things running in the data plane and around API can be obtained by Customers on their own. It could work like this:
+
+![h](assets/hypershift-cus.png)
+
+This is not different to GKE/EKS cases, where, the customer has to deploy on their own kube-state-metrics, node-exporter, cadvisor etc and configure Open Metrics compatible solution to scrape those.
+
+As the Red Hat data plane, we used to offer this out-of-the-box ,so we should maintain this practice. In simple form we could use CMO without PM, but with all exporters and optional UWM. There is a slight tweak where UWM has to know scrape those data-plane KSM, node exporter, cadvisor.
+
+Our team also develops [Monitoring Stacks](https://github.com/openshift/enhancements/pull/866) which aims to do similar yet allows greater scalability and flexibility due to independent versioning and OLM installation model.
+
+Yet, CUS can use any other DIY or just simple Prometheus/Prometheus Operator or vendor (e.g Grafana Cloud or anything else too) to do similar.
```

Comment:
This whole passage is very confusing to me. Please clearly outline what the Hypershift product needs to do in order to provide customers with metrics, alerting and enable HPA/VPA to work. Today, we simply deploy the monitoring operator. Will this change or is there anything else that needs to be done?